box-share-sdk/src/main/res/layout/view_collaborators_initial.xml

```diff
@@ -4,18 +4,15 @@
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:orientation="vertical"
-    android:layout_marginTop="32dp"
-    android:layout_marginBottom="48dp"
     xmlns:android="http://schemas.android.com/apk/res/android">
     <TextView
         android:id="@+id/invite_collaborator_initials_list_header"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:textAppearance="@style/TextAppearance.AppCompat.Subhead"
-        android:textColor="@color/box_sharesdk_collaborators_color"
+        android:textColor="#404040"
```

Comment:
for these values it's better practice to setup a value in the colors.xml so that we can more easily change the style throughout later.